examples/with-reason-react/package.json

```diff
@@ -18,7 +18,7 @@
     "reason-react": "^0.5.3"
   },
   "devDependencies": {
-    "bs-platform": "^4.0.17",
+    "bs-platform": "^7.2.2",
```

Comment:
could you please explain why you upgraded this package?

Reply:
[In the 'Install and bootstrap packages' phase of the Circle CT 'test-node9' target, the bs-platform build fails](https://circleci.com/gh/jaredpalmer/razzle/1313). I'm not sure why, but it was fixed by upgrading the bs-platform version. Another solution is to drop non LTS versions of the tests. Problem case of the module currently being used by razzle is get-caller-file, which prevents the build from breaking through the --ignore-engines flag, but it doesn't seem to be the underlying solution. https://github.com/stefanpenner/get-caller-file/issues/18